index.html

```diff
@@ -199,7 +199,8 @@ <h3>
                         Requirements for joining the team
                     </h3>
                     <ul>
-                        <li>...</li>
+                        <li>You should feel dedication to work on the docs regularly and take responsibility.</li>
```

Comment:
You should be able to dedicate time to working on the docs regularly and take responsibility for providing effective documentation for the community.